tool/tctl/common/lock_command_test.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/jonboulle/clockwork"
+	"github.com/stretchr/testify/require"
+
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/lib/config"
+)
+
+func TestLocks(t *testing.T) {
+	dynAddr := newDynamicServiceAddr(t)
+	fileConfig := &config.FileConfig{
+		Global: config.Global{
+			DataDir: t.TempDir(),
+		},
+		Proxy: config.Proxy{
+			Service: config.Service{
+				EnabledFlag: "true",
+			},
+			WebAddr: dynAddr.webAddr,
+			TunAddr: dynAddr.tunnelAddr,
+		},
+		Auth: config.Auth{
+			Service: config.Service{
+				EnabledFlag:   "true",
+				ListenAddress: dynAddr.authAddr,
+			},
+		},
+	}
+
+	timeNow := time.Now().UTC()
```

Comment:
I wouldn't use `time.Now()` at all.

Just use `NewFakeClock` instead of `NewFakeClockAt` and use `clock.Now()` instead.

Reply:
I wanted to be explicit because there was some discussion on the `clockwork` repo about having it return a different time for `now` when called multiple times so I felt this way would hopefully be more robust over time.

Reply:
Did you try calling `Now` multiple times?

> NewFakeClock returns a FakeClock implementation which can be manually advanced through time for testing. 

The fake clock doesn't advance unless you advance it. Whether you set the initial time or you let the library set the initial time doesn't change that behavior.

Reply:
Passing in the `NewFakeClock` instead of this approach causes issues with the SSL handshake in the test.... I'll continue to dig in and find out why. But my guess would be that the Jan 2 2006 is too long ago...

Reply:
Yep that's exactly what it is, so it's going to have to stay like this.